data/projects/optimize_caching.md

```diff
@@ -1,28 +1,46 @@
-This project involves improving coala's performance by implementing caching
-strategies. One example is to use git-like approach at file-collection level
-to not explore directories that haven't been modified. Caching per section can
-also be implemented so sections without issues don't rerun on all files where
-issues in other sections occurred.
+Currently, coala supports caching of project files. This was a big performance
+improvement over running code analysis for all files, even if most files
+are unchanged since the last run. But coala is still rather slow and could be
+much faster.
 
-Own performance optimization ideas are highly encouraged!
+In this project, you're to improve on the current strategies. Some enhancements
+such as section-level caching, globbing improvements to not expand on
+unmodified directories, and others have been preliminarily explored. Please
+refer to the related issues to learn more about these.
 
+The student is also highly encouraged to come up with new solutions that may
+increase performance even further.
 
 #### Milestones
 
-##### GSOC 2017 COMMUNITY BONDING
+##### PREPARATION/BONDING
 
-* The applicant has determined a number of circumstances where coala needlessly
-checks unmodified files.
-* The applicant has created testcases to validate performance improvements
-* A cEP describing the changes is merged.
+* The applicant has identified at least 3 other performance bottlenecks that can be  
+  improved. These can be smaller improvements, focussing on one particular  
```

Comment:
focussing and focusing are both correct, but holy god, the first one sounds so weird